hack/.golint_failures

```diff
@@ -3,11 +3,7 @@ cmd/hyperkube
 cmd/kube-apiserver/app
 cmd/kube-controller-manager/app
 cmd/kube-proxy/app
-cmd/kubeadm/app
 cmd/kubeadm/app/apis/kubeadm/v1alpha3
```

Comment:
how come this isn't removed?

Reply:
lots of auto-generated funcs under v1alpha3 fail golint, should we also fix them ? :
>
>Errors from golint:
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:35:1: exported function Convert_v1alpha3_InitConfiguration_To_kubeadm_InitConfiguration should have comment or be unexported
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:35:6: don't use underscores in Go names; func Convert_v1alpha3_InitConfiguration_To_kubeadm_InitConfiguration should be ConvertV1alpha3InitConfigurationToKubeadmInitConfiguration
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:42:1: exported function Convert_kubeadm_InitConfiguration_To_v1alpha3_InitConfiguration should have comment or be unexported
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:42:6: don't use underscores in Go names; func Convert_kubeadm_InitConfiguration_To_v1alpha3_InitConfiguration should be ConvertKubeadmInitConfigurationToV1alpha3InitConfiguration
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:51:6: don't use underscores in Go names; func Convert_v1alpha3_JoinConfiguration_To_kubeadm_JoinConfiguration should be ConvertV1alpha3JoinConfigurationToKubeadmJoinConfiguration
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:103:6: don't use underscores in Go names; func Convert_kubeadm_JoinConfiguration_To_v1alpha3_JoinConfiguration should be ConvertKubeadmJoinConfigurationToV1alpha3JoinConfiguration
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:131:1: exported function Convert_v1alpha3_ClusterConfiguration_To_kubeadm_ClusterConfiguration should have comment or be unexported
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:131:6: don't use underscores in Go names; func Convert_v1alpha3_ClusterConfiguration_To_kubeadm_ClusterConfiguration should be ConvertV1alpha3ClusterConfigurationToKubeadmClusterConfiguration
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:174:1: exported function Convert_v1alpha3_UnifiedControlPlaneImage_To_kubeadm_UseHyperKubeImage should have comment or be unexported
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:174:6: don't use underscores in Go names; func Convert_v1alpha3_UnifiedControlPlaneImage_To_kubeadm_UseHyperKubeImage should be ConvertV1alpha3UnifiedControlPlaneImageToKubeadmUseHyperKubeImage
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:190:1: exported function Convert_kubeadm_ClusterConfiguration_To_v1alpha3_ClusterConfiguration should have comment or be unexported
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:190:6: don't use underscores in Go names; func Convert_kubeadm_ClusterConfiguration_To_v1alpha3_ClusterConfiguration should be ConvertKubeadmClusterConfigurationToV1alpha3ClusterConfiguration
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:228:1: exported function Convert_v1alpha3_HostPathMount_To_kubeadm_HostPathMount should have comment or be unexported
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:228:6: don't use underscores in Go names; func Convert_v1alpha3_HostPathMount_To_kubeadm_HostPathMount should be ConvertV1alpha3HostPathMountToKubeadmHostPathMount
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:237:1: exported function Convert_kubeadm_HostPathMount_To_v1alpha3_HostPathMount should have comment or be unexported
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:237:6: don't use underscores in Go names; func Convert_kubeadm_HostPathMount_To_v1alpha3_HostPathMount should be ConvertKubeadmHostPathMountToV1alpha3HostPathMount
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:246:6: don't use underscores in Go names; func convertSlice_v1alpha3_HostPathMount_To_kubeadm_HostPathMount should be convertSliceV1alpha3HostPathMountToKubeadmHostPathMount
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:260:6: don't use underscores in Go names; func convertSlice_kubeadm_HostPathMount_To_v1alpha3_HostPathMount should be convertSliceKubeadmHostPathMountToV1alpha3HostPathMount
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:275:1: exported function Convert_v1alpha3_LocalEtcd_To_kubeadm_LocalEtcd should have comment or be unexported
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:275:6: don't use underscores in Go names; func Convert_v1alpha3_LocalEtcd_To_kubeadm_LocalEtcd should be ConvertV1alpha3LocalEtcdToKubeadmLocalEtcd
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:310:1: exported function Convert_kubeadm_LocalEtcd_To_v1alpha3_LocalEtcd should have comment or be unexported
cmd/kubeadm/app/apis/kubeadm/v1alpha3/conversion.go:310:6: don't use underscores in Go names; func Convert_kubeadm_LocalEtcd_To_v1alpha3_LocalEtcd should be ConvertKubeadmLocalEtcdToV1alpha3LocalEtcd

Reply:
No, probably not